platform/android/java/app/build.gradle

```diff
@@ -184,6 +196,12 @@ android {
         }
     }
 
+    flavorDimensions 'flavor'
```

Comment:
The dimension is used to characterize and group the flavors. For this use-case the dimension could be `languages`; though given you mentioned using `standard` as the alternative, maybe another term should be used.

Reply:
Yeah, in the download card we're using the term `standard` to differentiate from the `mono` (.NET) version of Godot.

![image](https://github.com/godotengine/godot/assets/3903059/e5afcfc8-7e90-48bd-bb3d-fe36104912b2)

In the website these are referred to as the `flavor`:

https://github.com/godotengine/godot-website/blob/1fee2ab1083b4d6b46bb84fb8498ca591f959b0b/_includes/articles/download_card.html#L9

So I'm not sure what to call the dimension if we don't like `flavor`. How about `edition`?

Reply:
`edition` sounds good!